plugins/inputs/rethinkdb/rethinkdb_server.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 type Server struct {
-	Url          *url.URL
+	URL          *url.URL
```

Comment:
This one might also be exposed to `toml` through the `RethinkDB` struct.

Reply:
`Server` is only created manually. `RethinkDB` struct uses array of strings.

```
type RethinkDB struct {
	Servers []string
}
```